Configuration/StandardSequences/python/RawToDigi_cff.py

```diff
@@ -38,6 +38,15 @@
 
 from L1Trigger.Configuration.L1TRawToDigi_cff import *
 
+from CondFormats.TotemReadoutObjects.TotemDAQMappingESSourceXML_cfi import *
+TotemDAQMappingESSourceXML.mappingFileNames.append("CondFormats/TotemReadoutObjects/xml/ctpps_210_mapping.xml")
+
+from EventFilter.TotemRawToDigi.totemTriggerRawToDigi_cfi import *
+totemTriggerRawToDigi.rawDataTag = cms.InputTag("rawDataCollector")
+
+from EventFilter.TotemRawToDigi.totemRPRawToDigi_cfi import *
+totemRPRawToDigi.rawDataTag = cms.InputTag("rawDataCollector")
```

Comment:
please follow the same pattern as for other Digis (imports listed first, assignment of rawDataCollector is done below)

I prefer that TotemDAQMappingESSourceXML does not appear in this file.
If it's a part required for totemRPRawToDigi, define totemRPRawToDigi_cff which loads TotemDAQMappingESSourceXML_cfi